connect/config.py

```diff
@@ -17,36 +17,36 @@ def __init__(
             api_url=None,
             api_key=None,
             products=None,
-            file=None
+            filename=None
```

Comment:
why to rename to filename? People will introduce there config.json, why not file? Shall we differenciate also filepath?

Reply:
Because PyCharm's code inspection raises a warning because 'file' is used as a standard type on Python. In this PR, I just made the required changes to the code so it can be inspected by PyCharm without any warnings.

Reply:
can you suggest better name than filename? I don't like it 100% becouse people also will put path